.github/dependabot.yml

```diff
@@ -3,26 +3,53 @@ updates:
 - package-ecosystem: pip
   directory: /
   schedule:
-    interval: weekly
+    interval: monthly
     # Add assignees
   assignees:
   - samster25
   - jaychia
+  groups:
+    # Python updates are mainly dev-only, so we can group them together.
+    all:
+      patterns:
+      - "*"
 
 - package-ecosystem: cargo
   directory: /
   schedule:
-    interval: weekly
+    interval: monthly
     # Add assignees
   assignees:
   - samster25
```

Comment:
can we assign some other folks besides jay and I? it is pretty noisy for us

Reply:
Assignees is not a required field, so we can leave them blank and oncall can assign it to themselves?

Reply:
SGTM